### DISCUSSION TITLE
Remove the "#" before reading aloud to make the speech content more natural

### DISCUSSION THREAD

**Discussion by Hunter6324** [open-webui/open-webui#7317] [Ideas]
The current reading function is to directly hand over the entire content of the large model's answers to a third party (Edge browser) to generate speech, and the "#" in the text will be read directly in the speech, such as "1. ## XXXX”。
If you deal with it, removing the "#" in the content will make the resulting voice more natural, after all, we don't directly shout "#" in dialogue, and the "#" in the text is only used as a format symbol.

**Comment by Simi5599** [comment]
Maybe this is related? #5612

I implemented this on my side and it works
